docs/wiki/User Guide/Configurations.md

```diff
@@ -232,6 +232,12 @@ We are still trying to improve cruise control. And following are some configurat
 | metric.reporter.topic                     | String | N         | "__CruiseControlMetrics"                                   | The exact topic name from which the sampler should be consuming the interested metrics from.                                             |
 | metric.reporter.sampler.group.id          | String | N         | 60,000                                                     | The consumer group id to use for the consumers to consume from the Kafka cluster.                                                        |
 
+### PrometheusMetricSampler configurations
```

Comment:
Can we update the table of contents with this new title?

Reply:
Done.